test/reference/json/Buildings/Controls/OBC/CDL/Continuous/PIDWithReset.json

```diff
@@ -4948,12 +4948,12 @@
                                       }
                                     ],
                                     "textString": "\"PI\"",
-                                    "visible": "([object Object])",
-                                    "lineColor": {
+                                    "textColor": {
                                       "r": 0,
                                       "g": 0,
                                       "b": 0
                                     },
+                                    "visible": "([object Object])",
```

Comment:
Should it be changed in different PR?

Reply:
created a new issue to address this (https://github.com/lbl-srg/modelica-json/issues/212)